src/components/Pokemonlist.js

```diff
@@ -0,0 +1,55 @@
+import React, {useEffect, useState} from "react";
+import axios from "axios";
+import Pokemon from "./Pokemon";
+
+function Pokemonlist() {
+    const [limit, setLimit] = useState(20)
+    const [page, setpage] = useState(0)
+    const [pokedex, setPokedex] = useState([])
+
+    useEffect(() => {
+        async function fetchPokedex() {
+            try {
+                const result = await axios.get(`https://pokeapi.co/api/v2/pokemon?limit=${limit}&offset=${(page * 20)}`)
+                setPokedex(result.data)
+            } catch (error) {
+                console.error(error)
+            }
+        }
+
+        void fetchPokedex()
+    }, [page, limit])
+
+    return (
+        <>
+            <div className='outer-nav-container'>
+            <nav>
+                {Math.ceil(pokedex.count / limit)} pages
+                <div className='page-navigation'>
+                    <button type="button" disabled={page < 1 && true} onClick={() => setpage(page - 1)}>last page</button>
```

Comment:
` && true` is altijd overbodig
```suggestion
                    <button type="button" disabled={page < 1} onClick={() => setpage(page - 1)}>last page</button>
```